insights/specs.py

```diff
@@ -257,12 +257,14 @@ def docker_container_ids(broker):
 module = sf.listdir("/sys/module", name="module")
 modinfo = sf.foreach(module, "/usr/sbin/modinfo %s", name="modinfo")
 modprobe_conf = sf.simple_file("/etc/modprobe.conf", name="modprobe_conf", alias="modprobe.conf")
-sysconfig_mongod = sf.glob_file(["etc/sysconfig/mongod",
-                                 "etc/opt/rh/rh-mongodb26/sysconfig/mongod"],
+sysconfig_mongod = sf.glob_file(["/etc/sysconfig/mongod",
```

Comment:
Better reorder the specs with alphabetical order.

Reply:
Yes, sure.